templates/frontend/pages/userLogin.tpl

```diff
@@ -52,39 +52,40 @@
 						<input class="form-control" type="text" name="username" id="username" value="{$username|escape}" maxlength="32" required>
 					</div>
 
-					<div class="form-group password">
+					<div class="form-group">
 						<label for="password">
 							{translate key="user.password"}
 							<span class="required">*</span>
 							<span class="sr-only">
 								{translate key="common.required"}
 							</span>
 						</label>
-
 						<input class="form-control" type="password" name="password" id="password" value="{$password|escape}" password="true" maxlength="32" required>
-						<a href="{url page="login" op="lostPassword"}">
-							{translate key="user.login.forgotPassword"}
-						</a>
-					</div>
 
-					<div class="custom-control custom-checkbox remember__checkbox">
-						<input class="custom-control-input" type="checkbox" name="remember" id="remember" value="1" checked="$remember">
-						<label class="custom-control-label" for="remember">
-							{translate key="user.login.rememberUsernameAndPassword"}
-						</label>
+						<div class="custom-control custom-checkbox">
+							<input class="custom-control-input" type="checkbox" name="remember" id="remember" value="1" checked="$remember">
+							<label class="custom-control-label" for="remember">
+								{translate key="user.login.rememberUsernameAndPassword"}
+							</label>
+						</div>
 					</div>
 
-					<div class="form-group form-group-buttons">
+					<div class="form-group">
 						<button class="btn btn-primary" type="submit">
 							{translate key="user.login"}
 						</button>
 
 						{if !$disableUserReg}
-							{capture assign=registerUrl}{url page="user" op="register" source=$source}{/capture}
-							<a href="{$registerUrl}" class="register btn btn-secondary">
+							{capture assign="registerUrl"}{url page="user" op="register" source=$source}{/capture}
+							<a href="{$registerUrl}" class="btn btn-secondary">
 								{translate key="user.login.registerNewAccount"}
 							</a>
 						{/if}
+
+						<br><br>
```

Comment:
Typically I would recommend adding spacing through CSS rather than `<br>` tags, because it can be more easily overridden or modified.

Reply:
I was trying to avoid overly-specific styles (especially if it’s just something that appears in this particular context) -- I will replace the two line breaks by nesting the links in their own paragraph, which is still semantically-correct.